client/public/manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "SOF",
+  "name": "Smart office management",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "faviconordina.ico",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
     {
-      "src": "logo192.png",
+      "src": "ordinaFavicon.png",
```

Comment:
The naming is not consistent. The next one is OrdinaFavicon with caps and they are of different resolutions. Neither of them are favicons, they are application icons, so you might want to use ordinaLogo192 or ordinaIcon192 and I would also not mind if it is just kept to logo192